index.js

```diff
@@ -18,10 +20,13 @@ module.exports = function (opts) {
 			return;
 		}
 
+		var target = path.relative(opts.abspath, file.path);
```

Comment:
That's what I'm saying. There's no reason `vulcanize` should only expect a relative path.